### ISSUE TITLE
Records without an 008 field crashing on index from .mrc file.

### ISSUE THREAD

**Issue by kelynch** [projectblacklight/blacklight-marc#32]
When running `rake solr:marc:index MARC_FILE="some_marc_records.mrc"`, records without an 008 field crash when extract_marc is called to set the value of formatCode, at https://github.com/projectblacklight/blacklight-marc/blob/dc32e02c6003d3ef1473ebb73ac9f6324035c171/lib/blacklight/marc/indexer/formats.rb#L142.  

See full stacktrace:

```
** Invoke solr:marc:index (first_time)
** Invoke solr:marc:index:work (first_time)
** Invoke solr:environment (first_time)
** Execute solr:environment
** Execute solr:marc:index:work
2016-03-11T09:17:48-05:00  INFO    Traject::SolrJsonWriter writing to 'http://127.0.0.1:8983/solr/blacklight-core/update/json' in batches of 100 with 1 bg threads
2016-03-11T09:17:48-05:00  INFO    Indexer with 1 processing threads, reader: Traject::MarcReader and writer: Traject::SolrJsonWriter
2016-03-11T09:17:48-05:00 ERROR Unexpected error on record id `76/76` at file position 1
    while executing (to_field format at /$LOCAL_DIRECTORY/blacklight/app/models/marc_indexer.rb:23)
    Exception: NoMethodError: undefined method `extract_marc' for Blacklight::Marc::Indexer::Formats::FormatMap:Module
    /$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/blacklight-marc-6.0.0/lib/blacklight/marc/indexer/formats.rb:143:in `map_leader'

2016-03-11T09:17:48-05:00 ERROR Unexpected error on record id `222/222` at file position 3
    while executing (to_field format at /$LOCAL_DIRECTORY/blacklight/app/models/marc_indexer.rb:23)
    Exception: NoMethodError: undefined method `extract_marc' for Blacklight::Marc::Indexer::Formats::FormatMap:Module
    /$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/blacklight-marc-6.0.0/lib/blacklight/marc/indexer/formats.rb:143:in `map_leader'

2016-03-11T09:17:48-05:00 ERROR Unexpected error on record id `293/293` at file position 4
    while executing (to_field format at /$LOCAL_DIRECTORY/blacklight/app/models/marc_indexer.rb:23)
    Exception: NoMethodError: undefined method `extract_marc' for Blacklight::Marc::Indexer::Formats::FormatMap:Module
    /$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/blacklight-marc-6.0.0/lib/blacklight/marc/indexer/formats.rb:143:in `map_leader'

rake aborted!
NoMethodError: undefined method `extract_marc' for Blacklight::Marc::Indexer::Formats::FormatMap:Module
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/blacklight-marc-6.0.0/lib/blacklight/marc/indexer/formats.rb:143:in `map_leader'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/blacklight-marc-6.0.0/lib/blacklight/marc/indexer/formats.rb:179:in `block in get_format'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer/step.rb:144:in `call'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer/step.rb:144:in `execute'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer.rb:347:in `block (2 levels) in map_to_context!'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer.rb:401:in `log_mapping_errors'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer.rb:346:in `block in map_to_context!'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer.rb:340:in `each'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer.rb:340:in `map_to_context!'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/indexer.rb:475:in `block (2 levels) in process'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/traject-2.3.0/lib/traject/thread_pool.rb:108:in `block in maybe_in_thread_pool'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:348:in `call'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:348:in `run_task'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:337:in `block (3 levels) in create_worker'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:320:in `loop'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:320:in `block (2 levels) in create_worker'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:319:in `catch'
/$RVM_LOCATION/.rvm/gems/ruby-2.2.1@blacklight/gems/concurrent-ruby-1.0.1/lib/concurrent/executor/ruby_thread_pool_executor.rb:319:in `block in create_worker'
Tasks: TOP => solr:marc:index => solr:marc:index:work
```

**Comment by magibney** [comment]
addressed by pull request #37

**Comment by barmintor** [comment]
Fixed in #37 and released in v6.2.0
